.github/workflows/update_org_members.yml

```
@@ -0,0 +1,28 @@
+# GitHub Actions workflow for auto-inviting org members to human intervention.
+#
+# The file coordinate_package_policies.json contains the the thresholds for
+# triggering addition for each of the coordinate packages.
+
+name: Invite organzation members based on merged PRs
+
+on:
+  schedule:
+    # Run once a day at 22:14 UTC
+    - cron: '14 22 * * *'
+
+jobs:
+  invite_members:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+
+    - name: Install dependencies
+      run: pip install github3.py
+
+    - name: Perform copy
+      env:
+        ORG_INVITE_TOKEN: ${{ secrets.ORG_INVITE_TOKEN }}
```

I just read through https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes now and I *think* the token only needs `repo:invite` powers. So should probably say that here:
```suggestion
        ORG_INVITE_TOKEN: ${{ secrets.ORG_INVITE_TOKEN }}  # this token needs "repo:invite" permissions
```

---

Are you sure it is `repo:invite`? This is sending people an invite to the astropy GitHub org, not to a specific repo....

---

Ah, good point!  I think it's `write:org` now that you point that out @mwcraig, since that says "write access to organization membership".  I guess it might need to be both `write:org` and `repo:invite` although I'm not really sure if `repo:invite` means just for "collaborators" or if it includes teams that touch that repo.

I wonder if we need to test this in a sandbox org or something?  Or we can just guess and see if it fails the first time it actually tries to add someone...

```suggestion
        ORG_INVITE_TOKEN: ${{ secrets.ORG_INVITE_TOKEN }} # this token needs "write:org" and maybe "repo:invite" or "admin:org" permissions TODO: fix this with the right answer once it's tested
```